src/commands/buffer.rs

```diff
@@ -418,12 +418,20 @@ pub fn outdent_line(app: &mut Application) -> Result {
             let mut space_char_count = 0;
 
             // Check for leading whitespace.
-            for character in content.chars().take(tab_content.chars().count()) {
-                if character == ' ' {
-                    space_char_count += 1;
-                } else {
-                    // We've run into a non-whitespace character; stop here.
-                    break;
+            if tab_content.chars().next().unwrap() == '\t' {
+                // We're looking for a tab character.
+                if content.chars().next().unwrap() == '\t' {
+                    space_char_count = 1;
+                }
+           	} else {
+           		// We're looking for spaces.
```

Comment:
This line and the one above it are using a mix of tabs and spaces, which is causing unaligned indentation.

Reply:
Sorry about that. Fixed.